Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -324,6 +348,19 @@ def _get_org_id(self, org_name):
             return org_uid
         raise Code42OrgNotFoundError(org_name)
 
+    def _get_legal_hold_matter_id(self, matter_name):
+        matter_id = self.get_legal_hold_matter(matter_name).get("legalHoldUid")
+        return matter_id
+
+    def _get_legal_hold_matter_membership_id(self, user_id, matter_id):
+        member_pages = self._get_sdk().legalhold.get_all_matter_custodians(legal_hold_uid=matter_id,
+                                                                           user_uid=user_id)
```

Comment:
Is this wrapping part from their formatter?

Reply:
no, I just did this to meet line length requirements